sormas-e2e-tests/src/test/resources/features/sanity/web/Contacts.feature

```diff
@@ -131,3 +131,23 @@ Feature: Contacts end to end tests
     And I am accessing the contacts from New Visit
     And I open Follow up Visits tab from contact directory
     Then I am validating the From and To dates displayed
+
+  Scenario: Create complex contact
```

Comment:
add also jira test case identifier

Reply:
I don't know why it doesn't show up here, but  I added test case identifier

Reply:
seems to be displayed now :D